infra/nncc/Makefile.arm32

```diff
@@ -9,6 +9,7 @@ CURRENT_DIR=$(shell pwd)
 BUILDFOLDER=build
 ARM32_FOLDER=arm32
 ROOTFS_ARM?=$(CURRENT_DIR)/tools/cross/rootfs/arm
+NNCC_CFG_OPTION_ADD?=
```

Comment:
I am not sure the usage of the varible, but how about name NNCC_CFG_OPTION_EXTRA or something if this is for a sort of extra option ?

Reply:
> I am not sure the usage of the varible,

This is to provide `additional` option from the user(caller) of this Makefile.arm32.

like
```
NNCC_CFG_OPTION_ADD=-DCMAKE_DO_THIS=THAT make -f infra/nncc/Makefile.arm32 all
```

`NNCC_CFG_OPTION_EXTRA` also looks OK. If @hyunsik-yoon also agrees I'll change it.

Reply:
@hyunsik-yoon PTAL

Reply:
`NNCC_CFG_OPTION_EXTRA` seems better to me. I thought that `ADD` meant adding some more cfg options from users. 😹

Reply:
> adding some more cfg options from users

Yes, it's adding more cfg options... anyway I'll revise this :)